autocards.py

```diff
@@ -1,438 +1,387 @@
         self.title = title
 
-        if per_paragraph:
+        if process_text_per_paragraph:
             print("Consuming text by paragraph:")
             for paragraph in tqdm(text.split('\n\n'),
                                   desc="Processing by paragraph",
                                   unit="paragraph"):
                 paragraph = paragraph.replace("\n", " ")
-                self._call_qg(paragraph, title)
+                self._call_question_generation_module(paragraph, title)
         else:
             print("Consuming text:")
             text = re.sub(r"\n\n*", ". ", text)
             text = re.sub(r"\.\.*", ".", text)
             text = self._sanitize_text(text)
-            self._call_qg(text, title)
+            self._call_question_generation_module(text, title)
 
-    def consume_user_input(self, title="untitled user input"):
-        "Take user input and create qa pairs"
+    def convert_user_input_into_question_answering_pairs(self, title="untitled user input"):
+        """Take user input and create qa pairs"""
         user_input = input("Enter your text below then press Enter (press\
  enter twice to validate input):\n>")
         user_input = user_input.strip()
 
         print("\nFeeding your text to Autocards...")
         user_input = self._sanitize_text(user_input)
-        self.consume_var(user_input, title, per_paragraph=False)
+        self.text_to_question_answering_pairs(user_input, title, process_text_per_paragraph=False)
         print("Done feeding text.")
 
-    def consume_pdf(self, pdf_path, per_paragraph=True):
-        "Take pdf file as input and create qa pairs"
+    def convert_pdf_into_question_answering_pairs(self, pdf_path, title="pdf file", per_paragraph=True):
+        """Take pdf pdf_file as input and create qa pairs"""
         if not Path(pdf_path).exists():
-            print(f"PDF file not found at {pdf_path}!")
+            print(f"PDF pdf_file not found at {pdf_path}!")
             return None
 
-        print("Warning: pdf parsing is usually of poor quality because \
-there are no good cross platform libraries. Consider using consume_textfile() \
-after preprocessing the text yourself.")
-        title = pdf_path.replace("\\", "").split("/")[-1]
-        raw = str(parser.from_file(pdf_path))
-        safe_text = raw.encode('utf-8', errors='ignore')
-        safe_text = str(safe_text).replace("\\n", "\n").replace("\\t", " ").replace("\\", "")
+        with open(pdf_path, "rb") as f:
+            pdf = pdftotext.PDF(f)
 
-        text = self._sanitize_text(safe_text)
+        text = "\n\n".join(pdf)
 
-        self.consume_var(text, title, per_paragraph)
+        self.text_to_question_answering_pairs(text, title, per_paragraph)
 
-    def consume_textfile(self, filepath, per_paragraph=False):
-        "Take text file as input and create qa pairs"
-        if not Path(filepath).exists():
+    def convert_text_file_into_question_answering_pairs(self, filepath, per_paragraph=True):
```

Comment:
Don't you think the methods are a bit verbose? Could we stick to the original signatures to support backwards compatibility, and if anyone doesn't understand the methods we add documentation?